app/goal_route.py

```diff
@@ -0,0 +1,138 @@
+from flask import Blueprint, request, jsonify, make_response, abort
+from app.models.goal import Goal
+from .models.task import Task
+from app import db
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+"""Wave05"""
+@goal_bp.route("", methods=["POST"])
+def create_goals():
+    resquest_body = request.get_json()
+    
+    if "title" in resquest_body:
+        goal = Goal(title=resquest_body["title"])
+    else:
+        return {"details": "Invalid data"}, 400
+    
+    db.session.add(goal)
+    db.session.commit()
+    
+    return {
+        "goal":
+            {
+            "id": goal.goal_id,
+            "title": goal.title
+            }
```

Comment:
This is a great place to use your to_json helper function:
```suggestion
        "goal": goal.to_json()
```